web/portal/platform/src/entities/BillableCall/Action/Export.tsx

```diff
@@ -0,0 +1,137 @@
+import { MoreMenuItem } from '@irontec/ivoz-ui/components/List/Content/Shared/MoreChildEntityLinks';
+import { StyledTableRowCustomCta } from '@irontec/ivoz-ui/components/List/Content/Table/ContentTable.styles';
+import { OutlinedButton } from '@irontec/ivoz-ui/components/shared/Button/Button.styles';
+import {
+  ActionFunctionComponent,
+  isSingleRowAction,
+  MultiSelectActionItemProps,
+} from '@irontec/ivoz-ui/router/routeMapParser';
+import _ from '@irontec/ivoz-ui/services/translations/translate';
+import CloudDownloadIcon from '@mui/icons-material/CloudDownload';
+import ErrorIcon from '@mui/icons-material/Error';
+import {
+  Box,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Tooltip,
+} from '@mui/material';
+import CircularProgress from '@mui/material/CircularProgress';
+import { useState } from 'react';
+import { useStoreActions } from 'store';
+
+import BillableCall from '../BillableCall';
+
+const Export: ActionFunctionComponent = (props: MultiSelectActionItemProps) => {
+  const { variant = 'icon' } = props;
+
+  const [open, setOpen] = useState(false);
+  const [error, setError] = useState(false);
+  const [errorMsg, setErrorMsg] = useState<string>('');
```

Comment:
Is this hint required?, I mean, when you set the initial value in useState, automatically infers such value from it.

Reply:
You're right. I'll change it